astro/src/content/blog/javascript-sdks.mdx

```diff
@@ -0,0 +1,177 @@
+---
+publish_date: 2023-10-17 
+title: Our New Vue, React and Angular SDKs
+description: JavaScript SDKs make integrating with FusionAuth easier
+authors: Dan Moore
+excerpt_separator: "{/* more */}"
+image: /img/blogs/hackfest/hackfest-fusionauth.png
+categories: Company
+tags: javascript, sdks, angular, vue, react, client libraries
+---
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import RemoteCode from "../../components/RemoteCode.astro";
+
+FusionAuth is, at its core, an architectural component of a larger application. I like to say "no one ever builds a full application with just FusionAuth". Sure, your application needs authentication and authorization, but there are other critical components. For example, the features of your application which make users *want* to register and log in.
+
+{/* more */}
+
+But here at FusionAuth, we want simple, secure authentication to be easy to add to any application. FusionAuth can run anywhere, but what use is that if it isn't easy to integrate? Applications which FusionAuth works with include:
+
+* JavaScript single pages applications (SPAs)
+* traditional webapps
+* commercial off the shelf applications
+* server side applications
+* APIs
+* mobile apps
+
+Pretty much anything which can use OpenID Connect (OIDC), Security Assertion Markup Language (SAML), or an HTTP based API should work. You can see our [list of quickstarts](/docs/quickstarts/) for integration examples.
+
+Recently, the team built software development kits (SDKs) to make it even easier for SPA developers. The SDKs support the following JavaScript front end frameworks:
+
+* [Angular](/docs/v1/tech/client-libraries/angular-sdk)
+* [React](/docs/v1/tech/client-libraries/react-sdk)
+* [Vue](/docs/v1/tech/client-libraries/vue-sdk)
+
+This post will cover the nitty gritty of using these SDKS, but first, let's discuss the difference between an SDK and a [client library](/docs/v1/tech/client-libraries/).
+
+## Client Libraries vs SDKs
+
+At FusionAuth, a client library is different from an SDK, even though many people in the industry use the terms interchangeably.
+
+### Client Libraries
+
+A client library is a wrapper over [FusionAuth APIs](/docs/v1/tech/apis/). Client libraries support all public FusionAuth API endpoints, and offer a convenient way to use that API in many software languages. At time of writing, there are eight supported languages, including TypeScript. There's also an OpenAPI definition, useful for viewing API changes over time or generating a client library for an unsupported language like C++ or Rust.
+
+If you want to compare a client library to a child's toy, it is like tinker toys. Engineers build a script or program using a client library to extend or configure FusionAuth. Some examples of tasks well suited to a client library:
```

Comment:
I don't get the analogy here. Maybe add that like tinker toys you have the building blocks but not the complete toy. The millennial in me also wouldn't reference a tinker toy...but that is for another day.

Reply:
I could use legos as the example too?